.travis.yml

```diff
@@ -17,3 +13,4 @@ before_script:
   - npm prune
 after_success:
   - npm run semantic-release
+  - npm run deploydocs
```

Comment:
also we have to run `"build"` before release, so add `"prepublish": "build"`. And instead of "deploydocs", use "deploy", otherwise the `demo/initials.js` file will be missing